legacy/src/main/java/org/opensearch/sql/legacy/request/SqlRequestFactory.java

```diff
@@ -82,6 +83,14 @@ private static Integer validateAndGetFetchSize(JSONObject jsonContent) {
         return fetchSize.orElse(0);
     }
 
+    private static List<String> parseRoutingIds(JSONArray array) {
```

Comment:
parseRoutingIds is not used any more.
parseSqlRequestFromUrl in this file is not used anywhere as well.